lib/Secrets/result.h

```diff
@@ -29,6 +29,8 @@ class Result {
         NoError = 0,
         UnknownError = 2,
         SerialisationError = 3,
+        DaemonError = 5,
```

Comment:
For symmetry with the Crypto::Result enum.  The first few values are fair generic, and I wanted to avoid potential confusion by aligning values where possible.

Reply:
The 1 value in particular is Pending from the Result::Code, which is why that one is "missing".

Reply:
Is this symmetry actually useful? I mean, the enums are exactly for handling via names, not numbers. So, the key abstraction of name => number is useless then.

Reply:
I disagree, it is useful, especially considering the very low cost of ensuring the symmetry.
There are a variety of reasons, but the most compelling is that the (secrets) storage error code is reported in the crypto result, as some crypto operations (e.g. generateStoredKey) also interact with the secrets storage (indirectly).  So it's useful to have the "generic" error codes match each other.